kolibri/core/assets/src/core-app/apiSpec.js

```diff
@@ -84,6 +83,10 @@ import * as contentNode from '../utils/contentNodeUtils';
 import attemptLogList from '../views/attempt-log-list';
 import interactionList from '../views/interaction-list';
 import examReport from '../views/exam-report';
+import kIndeterminateLinearIndicator from '../views/k-indeterminate-linear-indicator';
+import kDeterminateLinearIndicator from '../views/k-determinate-linear-indicator';
+import kIndeterminateCircularIndicator from '../views/k-indeterminate-circular-indicator';
```

Comment:
Maybe just make two two components, `kLinearLoader` and `kCircularLoader`, and give the linear one an `determinate` prop?